jaxrs/src/main/java/com/cerner/beadledom/jaxrs/PATCH.java

```diff
@@ -0,0 +1,19 @@
+package com.cerner.beadledom.jaxrs;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import javax.ws.rs.HttpMethod;
+
+/**
+ * Indicates that the annotated method responds to HTTP PATCH requests.
+ *
+ * @author John Leacox
```

Comment:
If you insist :) [bb8be52](https://github.com/cerner/beadledom/pull/36/commits/bb8be5203c6721fe2b3ef5d03c017bef69441e43)